File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
    int height;
    
    do
    {
    height = get_int("Height?\n");
    }
    while (height >8 || height < 1);
    
    for (int i = 0; i < height; i++)
    {
        // Print " " (n - 1 - i) times
        for (int d = height - 1; d > i; d--)
        {
            printf(" ");
        } 

        // Print '#' (i + 1) times
        for (int h = -1; h < i; h++)
        {
            printf("#");
        } 
        printf("\n");
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int

luigi = 0

while True:
    luigi = get_int("Height: ")
    if luigi > 0 and luigi < 9:
        break

for i in range(1, luigi + 1):
    block = i
    space = luigi - block
    print(" " * space, end="")
    print("#" * block, end="")
    print("  ", end = "")
    print("#" * block, end="")
    print()
